packages/cli-tool/src/add-ons/ui-framework/tailwind-css.ts

```diff
@@ -0,0 +1,100 @@
+import * as fs from 'fs';
+
+import { cli } from 'cli-ux';
+
+import runCommand from '../../helpers/child-process';
+import { replaceLine } from '../../helpers/file-editor';
+
+const NPM_PACKAGES = [
+  'tailwindcss@^3.1.4',
+  'postcss@^8.4.14',
+  'postcss-import@^14.1.0',
+];
+
+const installNpmPackages = (appName: string): Promise<void> => {
```

Comment:
What do you think of renaming it to`installNpmDevPackages` or `installDevDependencies` as this function installs the provided `appName` as dev dependencies?

Reply:
Yes, IMO it is quite similar so updated: d826a98 ;-)